azureiot.sln

```diff
@@ -152,11 +152,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SymmetricKeySample", "provi
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "InvokeDeviceMethod", "iothub\service\samples\getting started\InvokeDeviceMethod\InvokeDeviceMethod.csproj", "{A9058CEC-E0EB-4461-A1B0-4C6CB83A8AAC}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EnrollmentGroupX509Sample", "provisioning\service\samples\getting started\EnrollmentGroupX509Sample\EnrollmentGroupX509Sample.csproj", "{83170B36-375E-40A9-B25D-999072362626}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EnrollmentGroupSample", "provisioning\service\samples\getting started\EnrollmentGroupSample\EnrollmentGroupSample.csproj", "{83170B36-375E-40A9-B25D-999072362626}"
```

Comment:
Changed X509 to symmetric key because the type of authentication for this sample seems irrelevant, and this simplifies execution.